app/src/org/commcare/logging/AndroidLogSerializer.java

```diff
@@ -78,11 +79,12 @@ public void writeToDeviceReport(XmlSerializer serializer) throws IOException {
         serializer.startTag(DeviceReportWriter.XMLNS, "log_subreport");
 
         try {
-            if (isSingleLog) {
-                serializeLog(entry);
-            } else {
-                if (Logger._() != null) {
-                    Logger._().serializeLogs(this);

```

Comment:
Hm, I can't do that unless I make `AndroidLogger` abstract because it inherits those methods from `ILogger`, which is an interface. And it does look like j2me still makes use of the `serializeLogs` methods in `ILogger`, so those methods can't be totally removed. I could make the `AndroidLogger` implementations return `null` though, to indicate that they're unused. Not sure how I feel about that pattern, but open to it.